content/blog/deep_reinforcement_for_algorithmic_trading.md

```diff
@@ -5,8 +5,34 @@ draft = true
 date = 2022-02-05T00:00:00Z
 
 [extra]
-author = "Hanno Tauber"
+author = "Hanno Tauber, dmtrsflaco"
```

Comment:
@dmtrs Could you please change to match `Dmtrs Flaco Meng` as found #2 or anything you prefer but please remember to keep it consistent :D

Reply:
✅